specification/batch/data-plane/Microsoft.Batch/2017-09-01.6.0/BatchService.json

```diff
@@ -10912,6 +10915,26 @@
       },
       "title": "The execution constraints for a job."
     },
+    "TaskContainerSettings": {
+      "properties": {
+        "containerRunOptions": {
+          "type": "string",
+          "title": "Additional options to the container run command.",
+          "description": "These additional options are supplied as arguments to the \"docker create\" command, in addition to those controlled by the Batch Service."
+        },
+        "imageName": {
+          "type": "string",
+          "title": "The container image under which the task will run."
```

Comment:
Perhaps add a description explaining whether this is a container reference e.g. `docker.io/itowlson/hello-world:latest` or just the name e.g. `hello-world`?

Reply:
It would be helpful to add a specific example such as "...for example, 'docker.io/itowlson/hello-world:latest'." (or whatever format is correct)

Reply:
I think I've improved this